src/components/ListView.js

```diff
@@ -1,17 +1,36 @@
-import React from "react";
-import { useCollection } from "react-firebase-hooks/firestore";
-import { db } from "../lib/firebase";
+import React, { Fragment } from 'react';
+import { useCollection } from 'react-firebase-hooks/firestore';
+import { db } from '../lib/firebase';
+import { firestore } from 'firebase';
```

Comment:
This unused import should be removed. If we add a linter to the project such as [ESLint](https://eslint.org/) we could both encourage/enforce a single/double quote style for all code contributors, as well as raise editor errors when an unused import remains.